notebooks/adhoc/jovian_analysis/jovian_src/analysis_functions.py

```diff
@@ -46,24 +52,28 @@ class JovianAnalysisResult:
     p95_util_vs_gas_used: float = 0.0
     share_over_1_vs_used: float = 0.0
     blocks_with_gas_used: int = 0
+    # Target-based metrics
+    blocks_exceeding_target: int = 0
+    percentage_exceeding_target: float = 0.0
+    avg_target_utilization: float = 0.0
+    max_target_utilization: float = 0.0
 
 
 def perform_jovian_analysis(
     df: pl.DataFrame,
-    gas_limit: int,
     da_footprint_gas_scalars: List[int] = None,
     chain: str = "base",
     sampling_method: str = "top_percentile",
     start_date: str = None,
     end_date: str = None,
-    show_progress: bool = True
+    show_progress: bool = True,
+    eip1559_elasticity: float = 2.0
```

Comment:
elasticity is always `int`

Reply:
I feel uneasy about using a default value... different chains have different elasticities, e.g. OPM has 2, Base has 3. We should get the elasticity for each chain, at least the latest block, rather than using a default value.